test/extensions/filters/http/jwt_authn/all_verifier_test.cc

```diff
@@ -18,16 +18,75 @@ namespace HttpFilters {
 namespace JwtAuthn {
 namespace {
 
+const char kConfigTemplate[] = R"(
+providers:
+  example_provider:
+    issuer: https://example.com
+    from_headers:
+    - name: "x-example"
+      value_prefix: ""
+    forward_payload_header: "x-example-payload"
+    local_jwks:
+      inline_string: ""
+  other_provider:
+    issuer: https://other.com
+    from_headers:
+    - name: "x-other"
+      value_prefix: ""
+    forward_payload_header: "x-other-payload"
+    local_jwks:
+      inline_string: ""
+rules:
+- match:
+    path: "/"
+)";
+
+const std::string kExampleHeader{"x-example"};
+const std::string kOtherHeader{"x-other"};
```

Comment:
no static or global std::string, (use `constexpr absl::string_view` instead.

Reply:
Changed to constexpr char[]. string_view need to do few castings later.

Reply:
either is fine :)